tabify/fecHeatmapTabify.js

```diff
@@ -0,0 +1,62 @@
+import isEmpty from "lodash/isEmpty";
+
+/*
+ * Returns the Network Loss and Loss after FEC percentages for a given Source and Destination NSG combination in the given format
+ * [
+ *   {
+ *     "key_as_string": "2020-08-24T18:40:00.000Z",
+ *     "date_histo": 1598294400000,
+ *     "doc_count": 1,
+ *     "stat": "NetworkLoss",
+ *     "ColorValue": "0.0% - 0.499%"
+ *   },
+ *   {
+ *     "key_as_string": "2020-08-24T18:40:00.000Z",
+ *     "date_histo": 1598294400000,
+ *     "doc_count": 1,
+ *     "stat": "LossAfterFEC",
+ *     "ColorValue": "0.0% - 0.499%"
+ *   },
+ *   ...
+ * ]
+ */
+export default class FecHeatmapTabify {
+    
+    process(response) {
+        const aggregations = response && response.aggregations;
+        if (!isEmpty(aggregations)) {
+            const result = [];
+            const getFECHeatmapColorValue = (key) => {
```

Comment:
make this function a class function otherwise you create an instance every time process is called.

Reply:
Done.